frontends/php/include/test.inc.php

```diff
@@ -128,4 +137,128 @@ function add_to_testing_table($item) {
    $result = DBexecute($sql);
 
    return $result;
-}
\ No newline at end of file
+}
+
+/**
+ * Removes item from item_testing table 
+ *
+ * @param string $itemid   item to be removed
+ * 
+ * @return bool
+ */
+function remove_from_testing_table($itemid) {
+   $result = false;
+   try [
+      $result = DB::delete('item_testing', array('itemid'=>$itemid));
+   ] catch (Exception $e) {
+      $result = true;
+   }
```

Comment:
- Are square brackets really block delimiters as well in PHP?
- Shouldn't exception result in false return value?
- I think exception should at least be logged somehow. Would be nice if user would get some feedback about failure in UI as well.